java/test/src/main/java/test/Ice/classLoader/AllTests.java

```diff
@@ -134,7 +134,6 @@ public static void allTests(test.TestHelper helper, boolean collocated)
                     out.flush();
                     ConcreteClass cc = initial.getConcreteClass();
                     test(cc != null);
-                    test(classLoader.check("Test.ConcreteClass"));
```

Comment:
This corresponded to the failed first attempt without the default package prefix.